TGS3.json

```diff
@@ -0,0 +1,58 @@
+{
+  "documentation": [
+    "Place this file in the root of your repo in order to configure TGS3 per repository options",
+    "This file is not required, it merely configures changelog generation, static directories, and .dlls",
+    "All keys except the ones below are ignored by the service"
+  ],
+  "changelog": {
```

Comment:
Use the preaction framework :^)

Reply:
You're not wrong, I'm just laz... oh I thought of how to do this!

Reply:
Actually no, I'll clean this up after 3.1